### ISSUE TITLE
The interface's font is too large.

### ISSUE THREAD

**Issue by ksubramanyeshwara** [pieces-app/support#307]
### Software

Desktop Application

### Operating System / Platform

Linux

### Your Pieces OS Version

10.0.0

### Early Access Program

- [ ] Yes, this is related to an Early Access Program feature.

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

The recent interface update, specifically the increased font size in the answer section, limits the number of answers that can be displayed. 
![Screenshot from 2024-07-12 22-14-57](https://github.com/user-attachments/assets/3265af50-ffc6-4ab1-b6b8-397cd3979a9f)

**Comment by shivay-at-pieces** [comment]
Hey thanks for sharing this @ksubramanyeshwara, the team is already working on a few UI adjustments for the Pieces Copilot app. Will keep you updated.

Also We are aware that some of the Pieces services are currently down including the website and the update functionality. We’re very sorry for the inconvenience this may have caused. The team is working to resolve the issues as quickly as possible.

**Comment by mack-at-pieces** [comment]
Hi @ksubramanyeshwara - Thanks so much for bringing this to our attention. Do you have any adjusted scaling/resolution/font size settings for your Linux system? If so, this may be impacting the sizing of the Desktop app. If not, we can try and see what else may be causing this. Thanks!

**Comment by ksubramanyeshwara** [comment]
> Hi @ksubramanyeshwara - Thanks so much for bringing this to our attention. Do you have any adjusted scaling/resolution/font size settings for your Linux system? If so, this may be impacting the sizing of the Desktop app. If not, we can try and see what else may be causing this. Thanks!

No, I have not adjusted any scaling/resolution/font size in my system. After the recent update, the font size has become bigger. I would say the older one looks good. The only improvement to the older UI would be that when you minimize the sidebar, it should stretch to the entire width.
Thank You.

**Comment by mack-at-pieces** [comment]
Thanks @ksubramanyeshwara . Within the desktop app, I would recommend tapping "Ctrl+" or "Ctrl-" to adjust the zoom of the desktop application. Happy to hop on a quick video call to troubleshoot live if that's easier.

**Comment by choir27** [comment]
The user should be able to zoom out/in the Pieces onboarding screens.

Steps to reproduce:
1. install pieces os and desktop
2. go to any page in the onboarding process
3. `ctr` + `+` or `ctr` + `-`

If the user is able to zoom in/out in the main application, they should also be able to do the same in the onboarding screens, for both accessibility and consistency purposes.

![Screenshot 2024-08-02 174935](https://github.com/user-attachments/assets/a7b570cd-de25-48c6-8bf1-8f3479684c6b)

**Comment by ksubramanyeshwara** [comment]
> Thanks @ksubramanyeshwara . Within the desktop app, I would recommend tapping "Ctrl+" or "Ctrl-" to adjust the zoom of the desktop application. Happy to hop on a quick video call to troubleshoot live if that's easier.

If I use "Ctrl+" or "Ctrl-" then the whole UI will be zoomed in or zoomed out. for example,
When zoomed in : 
![Screenshot from 2024-08-08 18-56-31](https://github.com/user-attachments/assets/e54566df-e8b4-4043-b17f-4640c5ef32af)
When zoomed out : 
![Screenshot from 2024-08-08 18-55-19](https://github.com/user-attachments/assets/b029d834-8ba7-4b7b-b4a9-910993aa19b0)
So you need to check the section where we ask question.

**Comment by mack-at-pieces** [comment]
Hey @ksubramanyeshwara - Yesterday we released version 3.1.0 of the Desktop App. Can you please update and let me know if you are still experiencing issues with this? Thanks!

**Comment by hra42** [comment]
@mack-at-pieces @choir27 had a similar bug on stream. If you click on the actual Chat window you can't zoom, but when you click on the input bar it works fine.

**Comment by 3dyuval** [comment]
I am using `3200x2000` in 200% and confirm this looks the same. Chat  font is extremely large. All other fonts are tool large as well.
